### PR TITLE
Breakage report improvements

### PR DESCRIPTION
**Pull request by federicocappelli**
Task/Issue URL: https://app.asana.com/0/1205842942115003/1205692741026215/f
Tech Design URL: https://app.asana.com/0/1205842942115003/1205917507991804/f

**Description**:
This PR introduces a new version of BSK (PR: https://github.com/duckduckgo/BrowserServicesKit/pull/566) that allows iOS and macOS to share the same web implementation of the `ReportBrokenSite` page and introduce improvements to Pixels and the broken site report data.

Additional changes
- The old native `Report broken site` page has been removed
- Added 2 new Pixels https://app.asana.com/0/1205842942115003/1205859726054342/f
- Removed legacy PrivacyDashboard.storyboard localised strings

**Steps to test this PR**:
Access `Report broken site` via privacy dashboard
1. Open `Privacy Dashboard`
2. Navigate to `Report broken site` via the `website not working` link on top of the page
3. Try to toggle the protection state
4. Try to submit a broken site report

Access `Report broken site` via menu
1. Tap on the "Report Broken Site" menu item 
2. Try to toggle the protection state
3. Try to submit a broken site report

**Copy Testing**:

No new copy to test

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE
* [ ] iPhone 15

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16
* [ ] iOS 17

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

### CODE REVIEW

File: DuckDuckGo/PrivacyDashboard/BrokenSiteInfo.swift
Comment (federicocappelli) [inline DuckDuckGo/PrivacyDashboard/BrokenSiteInfo.swift:1]: file moved

File: DuckDuckGo/Base.lproj/Feedback.storyboard
Comment (federicocappelli) [inline DuckDuckGo/Base.lproj/Feedback.storyboard:1]: ReportBrokenWebsite part removed from this storyboard

File: DuckDuckGo/Base.lproj/Settings.storyboard
Comment (federicocappelli) [inline DuckDuckGo/Base.lproj/Settings.storyboard:1]: ReportBrokenWebsite segue removed and replaced with the PrivacyDashboardVC

File: Widgets/bg.lproj/Localizable.strings
Comment (federicocappelli) [inline Widgets/bg.lproj/Localizable.strings:1]: Note: All these new strings are coming from the VPN implementation, from some reasons they were not present. I paired with Bartek and this change is ok, it's just removing the right strings and adding the new ones

**Review by tomasstrba — APPROVED** [review]
LGTM! 👍 

broken site reporting works ✔️ 
turning off protections work ✔️

**Pull request merged**